scout/demo/643594.config.yaml

```diff
@@ -62,6 +63,7 @@ samples:
     mitodel_file: scout/demo/mitodel_test_files/slowlycivilbuck_lanes_1_sorted_md_mitodel.txt
     mt_bam: scout/demo/reduced_mt.bam
     vcf2cytosure: scout/demo/ADM1059A1.test.cgh
+    rna_sample_id: RNA1059A1
```

Comment:
I don't like this. It adds another level of complication to an already complex config file..

Reply:
Ok, good! Counter suggestions?

Reply:
Adding a sample mapping in the omics section of the config file?

Reply:
Like
```
omics_files:
    fraser: fraser.tsv
    outrider: outrider.tsv
    sample_identification:
        rna_sample_id: RNA1059A1
        sample_id: ACC1059A1
```
Possible, but it is a bit unfortunate with the files in "omics_files" then.

Reply:
Did you see my refactor suggestion for laters as well? https://github.com/Clinical-Genomics/scout/discussions/4817

Reply:
> Did you see my refactor suggestion for laters as well? #4817

I'm reading it now. I see your point but I still think we should keep this PR as simple as possible. I agree that this PR will not be a solution to the sample naming, and that we should have a better way to connect samples and id samples, but this can be done in a dedicated PR. It feels a bit rushed to me right now and I'd rather reason on a permanent solution instead of having something that we'll need to change later